plugin/src/main/kotlin/com/github/triplet/gradle/play/ModifyReleaseTask.kt

```diff
@@ -0,0 +1,43 @@
+package com.github.triplet.gradle.play
+
+import com.github.triplet.gradle.play.internal.PlayPublishPackageBase
+import com.github.triplet.gradle.play.internal.TrackType
+import org.gradle.api.tasks.TaskAction
+
+open class ModifyReleaseTask : PlayPublishPackageBase() {
+    init {
+        // Always out-of-date since we don't know what's changed on the network
+        outputs.upToDateWhen { false }
+    }
+
+    private var startCodes: List<Long>? = null
```

Comment:
Let's make this a local variable so we don't have to worry about mutability or methods being called in a specific order.